public/index.php

```diff
@@ -7,12 +7,14 @@
  * Time: 14:01
  */
 
+session_cache_limiter('private, must-revalidate');
+session_cache_expire(60);
```

Comment:
Why do you do that ? (It doesn't mean it's an error. I want to be sur you do this for the rights reasons)